src/sentry/api/endpoints/auth_config.py

```diff
@@ -24,6 +24,10 @@ class AuthConfigEndpoint(Endpoint, OrganizationMixin):
     # Disable authentication and permission requirements.
     permission_classes = []
 
+    def dispatch(self, request: Request, *args, **kwargs) -> Response:
+        self.determine_active_organization(request)
```

Comment:
So a couple of things going on.  The `get_active_organization` method is being changed into two pieces: `determine_active_organization` which occurs in the dispatch at the head of a request, and the `active_organization` property which is set as a result of that.

Reasoning being is that `get_active_organization` is called in many many places, but it had side effects as well.  This approach is more akin to a middleware, but I didn't got that far with refactoring due to the tangled dependencies (and need for one subclass to override it).